### PR TITLE
feat: resource hub header

### PR DESCRIPTION
**Pull request by jianwei1**
# Description

### Issue

The current header in watch needs to be updated into the resources hub header

[Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/37663178/todos/7439735805)

### Solution

- Update LocalAppBar inside of Header.tsx
- Create HeaderTabButtons component
- Use LaunchDarkly flags to show/hide buttons

# External Changes

_If you have made changes to external services, need to add additional values to Doppler, or need to add something new to the database, explain it here. This may include updates to third-party services, changes to infrastructure configuration, integration with external APIs, etc._

# Additional information

_Provide any additional information that might be useful to the reviewer in evaluating this pull request. This could include performance considerations,design choices, etc._

### CODE REVIEW

File: apps/watch/src/components/Header/assets/favicon.png
Comment (jianwei1) [inline apps/watch/src/components/Header/assets/favicon.png:1]: change to minimal-logo

File: apps/watch/src/components/HomePage/HomePage.tsx
Comment (jianwei1) [inline apps/watch/src/components/HomePage/HomePage.tsx:1]: rename to WatchHomePage

File: apps/watch/src/components/Header/assets/favicon.png
Comment (jianwei1) [reply apps/watch/src/components/Header/assets/favicon.png:1]: https://github.com/JesusFilm/core/pull/2815/commits/060d2b38797b9a64bd764fdab74303464dcc02a6

File: apps/watch/src/components/Header/Header.spec.tsx [hunk 1]
```diff
@@ -40,4 +40,11 @@ describe('Header', () => {
       queryByRole('button', { name: 'open header menu' })
     ).not.toBeInTheDocument()
   })
+
+  // should hide spacer
```
Comment (tataihono) [inline apps/watch/src/components/Header/Header.spec.tsx:44]: drop

**Review by tataihono — CHANGES_REQUESTED** [review]

**Review by tataihono — APPROVED** [review]

**Pull request merged**